app/assets/stylesheets/govuk_publishing_components/components/_table.scss

```diff
@@ -16,6 +16,19 @@ $table-row-even-background-colour: govuk-colour("light-grey", $legacy: "grey-4")
   color: $govuk-secondary-text-colour;
 }
 
+.govuk-table-filter {
+  display: none;
+}
+
+.govuk-table__message {
+  @include govuk-font($size: 19);
```

Comment:
I think you could remove this class entirely and use `govuk-body` in the template instead.

Reply:
Yes, good call. Ta.